packages/-ember-data/tsconfig.json

```diff
@@ -25,13 +25,14 @@
       "@ember-data/model/*": ["../model/addon/*"],
       "@ember-data/adapter": ["../adapter/addon"],
       "@ember-data/adapter/*": ["../adapter/addon/*"],
+      "@ember-data/adapter/error": ["../adapter/addon/error"],
```

Comment:
Do we make changes related to `@ember-data/adapter/error` in this PR?

Reply:
no, but this config is used when we run tests etc. so it is forced to see all the things. This should have been captured already with the `*` but I think vscode auto updated this based on an autocomplete.